demo-app/yarn.lock

```diff
@@ -0,0 +1,4063 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please remove this file, we are using NPM in the project and having yarn lock doesn't make sense.